src/navigation/BottomTabNavigator.tsx

```diff
@@ -39,6 +42,13 @@ const BottomTabNavigator = (): JSX.Element | null => {
   const renderDictionaryTabIcon = ({ focused }: { focused: boolean }) =>
     focused ? <BookIconWhite width={iconSize} height={iconSize} /> : <BookIconGrey width={iconSize} height={iconSize} />
 
+  const renderRepetitionTabIcon = ({ focused }: { focused: boolean }) =>
```

Comment:
Another tab Icon? I do not blame you as you did not do the design, but I think long term we should reconstruct this. The tab text is always cut off. :/ What do you think?

Reply:
Yes I think it's because my emulators screen is a little bit narrow. I also just left the comment to keep track of this. :)

Reply:
Happens on my real device as well though 🙈